Apps/Database/src/Delegates/DBProfileDelegate.cs

```diff
@@ -158,10 +158,10 @@ public int GeUnregisteredInvitedUsersCount()
         }
 
         /// <inheritdoc />
-        public int GetTodayLoggedinUsersCount()
+        public int GetLoggedInUsersCount(DateTime startDate)
         {
             int result = this.dbContext.UserProfile
-                .Count(u => u.LastLoginDateTime.HasValue && u.LastLoginDateTime.Value >= DateTime.UtcNow.Date);
+                .Count(u => u.LastLoginDateTime.HasValue && u.LastLoginDateTime.Value >= startDate);
```

Comment:
Not sure this solves the problem, Instead how about converting both dates to a passed timeZone. That way the calculation can still be made and know for sure that the date will match

Reply:
Then the database delegate would be handling date logic which i think its wrong. 
I'm pretty sure this should work though, the utc today is converted to vancouver today, and only utc records logged in after that will make to the count. using the scenario we talked about today would be 2020-03-30 08:00:00 utc

Reply:
The issue is that the delegate needs to do the query, which needs to account for the time zone. To make the delegate agnostic we can pass the timezone as a paramater